tests/testsuite.src/used_binaries.at

```diff
@@ -266,11 +269,17 @@ AT_CHECK([$GREP 'SECTION_END__PROC:' prog.c], [0], ignore, [])
 AT_CHECK([$GREP 'PARAGRAPH_00_l_4:'        prog.c], [0], ignore, [])
 AT_CHECK([$GREP 'PARAGRAPH_EX_l_7:'        prog.c], [0], ignore, [])
 AT_CHECK([$COBCRUN_DIRECT ./prog], [0], [bluBb END], [])
+
+AT_CAPTURE_FILE([prog.d])
 AT_CHECK([$COBC -I sub/copy prog.cob -ext=cpy -o prog.i -MF prog.d -MT "prog.c prog.h" -MT prog$COB_EXE_EXT -MT prog.$COB_OBJECT_EXT -MT prog.i  -fsyntax-only], [0], [], [])
 AT_CHECK([$GREP 'prog.c prog.h '        prog.d], [0], ignore, [])
 AT_CHECK([$GREP ' prog.i:'        prog.d], [0], ignore, [])
-AT_CHECK([$GREP 'sub/copy/PROC.cpy'        prog.d], [0], ignore, [])
-AT_CHECK([$GREP 'sub/PROCE.cpy'        prog.d], [0], ignore, [])
+AT_CHECK([$GREP 'sub/copy/PROC.cpy'    prog.d], [0], ignore, [],
+  # Previous test "failed" --> no entry with slash available, check backslash
+  AT_CHECK([$GREP 'sub\copy\PROC.cpy'  prog.d], [0], ignore, []))
```

Comment:
The additional output we now have in the testsuite output shows:

~~~
prog.d:
> prog.c prog.h prog.exe prog.o prog.i: \
>  prog.cob \
>  sub/copy\PROC.cpy \
>  sub\PROCE.cpy
> 
> prog.cob:
> sub/copy\PROC.cpy:
> sub\PROCE.cpy:
~~~

So "combined slashes"... yay.
I do remember that I've once coded a "use all slashes as the first one" or "replace all slashes by the 'native' one", possibly that was because of this issue (but long before the .d was re-added). I'll consider this, need to check the results of those.

Reply:
solved with latest updates